test/WhitelistNFT.t.sol

```diff
@@ -1,100 +1,105 @@
-// SPDX-License-Identifier: UNLICENSED
+// SPDX-License-Identifier: MIT
 pragma solidity ^0.8.13;
 
 import {Test, console} from "forge-std/Test.sol";
-import {WhitelistNFT} from "../src/WhitelistNFT.sol";
+import {BaseSetup} from "./BaseSetup.t.sol";
 
-contract WhitelistNFTTest is Test {
-    WhitelistNFT earlyAccessNFT;
-    address public recipient1 = 0x70997970C51812dc3A010C7d01b50e0d17dc79C8;
-    address public recipient2 = 0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC;
+contract WhitelistNFTTest is Test, BaseSetup {
 
-    function setUp() public {
-        earlyAccessNFT = new WhitelistNFT("WhitelistNFT", "KARROT");
-        uint256 balance = earlyAccessNFT.balanceOf(recipient1);
-        assertEq(balance, 0);
+    function setUp() public override {
+        BaseSetup.setUp();
     }
 
     //  -------- mint tests --------
 
     // test should pass if the NFT is minted to the recipient
     function test_MintToShouldPass() public {
-        uint256 tokenId = earlyAccessNFT.mintTo(recipient1);
```

Comment:
I find `earlyAccessNFT` better than `whietlistNFT`.

Lemme know what you think :)

Reply:
I prefer `earlyAccessNFT` as well but renamed to `whitelistNFT` because later is being used in contract name and filename. Let's change all `whitelistNFT` to `earlyAccessNFT` 💡

Reply:
cool